java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1893,6 +1893,11 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_makeCudfColumnView(
       j_null_count = 0;
     }
 
+    if (j_null_count == cudf::UNKNOWN_NULL_COUNT) {
```

Comment:
Since we are removing this constant, can we use this?
```suggestion
    if (j_null_count < 0) {
```

Reply:
We will eventually be removing the `cudf::UNKNOWN_NULL_COUNT` constant -- hopefully in this release since there should be no usage or reliance on it. If you still need the constant for the internal JNI transition, it may be best create your own constant.

Reply:
Agreed. I've changed this as per @ttnghia's suggestion.